### PR TITLE
Added port aliases testing.

### PR DESCRIPTION
**Pull request by antonpatenko**
Because of extending of the port_config.ini format to have a port alias for each port, one more field will be read from port_config.ini. This test evaluates if Redis DB has got the same ports aliases as ones in port_config.ini.

### CODE REVIEW

**Comment by lguohan** [comment]
can you also update the port_config.ini in the sonic-swss repo?

File: ansible/roles/test/tasks/sonic.yml [hunk 1]
```diff
@@ -62,8 +62,12 @@
   include: arpall.yml
   tags: arp
 
-### When calling this basic_route, Chip is setup with proper parameters and
-### SAI API used by syncd daemon are called. This test is mainly for validation purpose used by external vendors.
+- name: Test hardware configuration
```
Comment (lguohan) [inline ansible/roles/test/tasks/sonic.yml]: suggest to have a better name: test port alias configuration, and also rename the playbook name to port-alias.yml

**Comment by antonpatenko** [comment]
OK, thanks, I will.

**Comment by wangxin** [comment]
Hi @antonpatenko, this PR is to add ansible playbook based testing. If this test is still required, probably you can try to add a pytest based script. I am closing this PR for now.

**Pull request closed without merging**